edgelet/contrib/debian8/postinst

```diff
@@ -28,7 +29,7 @@ case "$1" in
             # Do this for debian systems which don't support FDNAMES
             touch /var/lib/iotedge/mgmt.sock
             touch /var/lib/iotedge/workload.sock
-            chown ${USER}:${GROUP} /var/lib/iotedge/mgmt.sock
+            chown ${AGENTUSER}:${GROUP} /var/lib/iotedge/mgmt.sock
```

Comment:
This opens up edgeAgent to access all APIs. Is there a plan to limit the APIs in iotedged?